src/main/cpp/t07_swap.cpp

```diff
@@ -19,5 +19,21 @@
 using namespace std;
 
 int t07_swap() {
+    int n = 0;
+    cin >> n;
+    int arr[n];
+    for (int i=0; i<n; i++) {
+        cin >> arr[i]; }
 
+    int ch = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'ch' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4432820)